src/components/SelectionView/SelectionView.css

```diff
@@ -0,0 +1,74 @@
+.selection-view {
+  display: flex;
+  padding: 2em;
+  min-height: calc(100vh - 11rem);
+  max-height: calc(100vh - 8rem);
+  flex-direction: column;
+  align-items: center;
+  align-content: center;
+  justify-content: center;
+}
+
+.advice-background {
+  background: linear-gradient(0deg, #f27a7d, #f7d486, #c5f9d7);
+  background-size: 400% 400%;
+  -webkit-animation: GradientAnimation 15s ease infinite;
+  -moz-animation: GradientAnimation 15s ease infinite;
+  animation: GradientAnimation 15s ease infinite;
+}
+
+.compliment-background {
+  background: linear-gradient(0deg, #c5f9d7, #f7d486, #f27a7d);
+  background-size: 400% 400%;
+  -webkit-animation: GradientAnimation 15s ease infinite;
+  -moz-animation: GradientAnimation 15s ease infinite;
+  animation: GradientAnimation 15s ease infinite;
+}
+
+.selection-topbar {
+  position: absolute;
+  top: 8rem;
+  right: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
```

Comment:
I know the styling was a lot of copy/paste from our homepage and very simple design, but I think it looks sleek! That 'get new *selection*' button and drop-shadow on the text gave it that extra "pop" we needed!